mifospay/src/main/res/layout/activity_receipt.xml

```diff
@@ -197,8 +197,8 @@
         android:layout_height="wrap_content"
         android:layout_gravity="bottom|end"
         android:layout_marginEnd="@dimen/fab_margin"
-        android:layout_marginRight="@dimen/fab_margin"
-        android:layout_marginBottom="@dimen/fab_margin"
+        android:layout_marginRight="@dimen/value_12dp"
+        android:layout_marginBottom="@dimen/value_80sp"
```

Comment:
Don't use `sp` for anything other than fonts. Sp stands for `Scalable pixels`, and these can vary (_be scaled_) as per the system font size preference of the user